Library/Homebrew/cmd/audit.rb

```diff
@@ -186,6 +186,7 @@ def audit_file
       [/^  keg_only/,                      "keg_only"],
       [/^  option/,                        "option"],
       [/^  depends_on/,                    "depends_on"],
+      [/^  (go_resource|resource)/,        "resources"],
```

Comment:
Mind using `(go_)?resource` or `(|go_)resource` in the regexp?

A true nitpick, but I think it's better to use `resource` as the label. Both `option` and `depends_on` can also have multiple instances, but they are not pluralized.